packages/react/src/components/TabContent/TabContent.js

```diff
@@ -19,6 +19,7 @@ const TabContent = props => {
   });
   return (
     <div
+      role="tabpanel"
```

Comment:
With the `role="tabpanel"`, do we need to include the tabIndex changes from the other PR? Seems like they come together as one distinct chunk to address the bug.

Reply:
Makes sense, fixed.